app/upgrade_handler.go

```diff
@@ -6,47 +6,32 @@ import (
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
-	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
-	"github.com/cosmos/cosmos-sdk/x/authz"
-	sdkauthz "github.com/cosmos/cosmos-sdk/x/authz"
-	"github.com/cosmos/cosmos-sdk/x/feegrant"
-	sdkfeegrant "github.com/cosmos/cosmos-sdk/x/feegrant"
+	crisistypes "github.com/cosmos/cosmos-sdk/x/crisis/types"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-
-	ibcconnectiontypes "github.com/cosmos/ibc-go/v2/modules/core/03-connection/types"
-
-	shieldtypes "github.com/certikfoundation/shentu/v2/x/shield/types"
 )
 
 const (
-	v230Upgrade = "Shentu-v230"
```

Comment:
If I understand correctly,  these two old handlers weren't needed because all nodes have upgraded to shield v2?

Reply:
They are already registered in the past + they should be only used for that specific upgrade